src/pages/Main/components/FilterModals/FilterModal.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import PriceRange from './FilterContents/PriceRange';
+import Convenients from './FilterContents/Convenients';
+import * as S from './FilterModal.styled';
+import CloseButton from 'react-bootstrap/CloseButton';
+import RoomOption from './FilterContents/RoomOption';
+import Button from 'react-bootstrap/Button';
+
+const FilterModal = ({
+  searchParams,
+  setSearchParams,
+  setIsFilterModalOpen,
+  setSelectedFilters,
+}) => {
+  const fixedMaxPrice = 1000000;
+  const fixedMinPrice = 0;
+  const [rangeMinValue, setRangeMinValue] = useState(fixedMinPrice);
+  const [rangeMaxValue, setRangeMaxValue] = useState(fixedMaxPrice);
+  const [selectedConvenients, setSelectedConvenients] = useState([]);
+  const [selectedRoomOption, setSelectedRoomOption] = useState({});
+
+  // console.log('최저가격', rangeMinValue);
+  // console.log('최고가격:', rangeMaxValue);
+  // console.log('편의시설', selectedConvenients);
+  // console.log('룸옵션', selectedRoomOption);
+
+  const handleRangeChange = (min, max) => {
+    setRangeMinValue(min);
+    setRangeMaxValue(max);
+    searchParams.set('price_max', max);
+    searchParams.set('price_min', min);
+  };
+
+  const handleCloseModal = () => {
+    setIsFilterModalOpen(false);
+  };
+
+  const handleConvenient = data => {
+    searchParams.delete('convenients');
+    data.forEach(list => {
+      searchParams.append('convenients', list);
+    });
+  };
+
+  const handleRoomOptionClick = (id, value) => {
+    setSelectedRoomOption({
+      ...selectedRoomOption,
+      [id]: value,
+    });
+    if (id === 'beds') {
+      searchParams.set('beds', value);
+    } else if (id === 'bedrooms') {
+      searchParams.set('bedrooms', value);
+    } else if (id === 'bathrooms') {
+      searchParams.set('bathrooms', value);
+    }
```

Comment:
```suggestion
      searchParams.set(id, value);
```

이 부분은 인자로 전달 받는 값을 넣어주면 되지 않을까요?